### PR TITLE
fix: reenable service feedstocks

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Added a ``news`` entry
* [x] Regenerated schema JSON if schema altered (`python conda_smithy/schema.py`)

### CODE REVIEW

**Comment by beckermr** [comment]
pre-commit.ci autofix

**Comment by beckermr** [comment]
@conda-forge/core @jaimergp this one is ready for review!

File: conda_smithy/configure_feedstock.py [hunk 1]
```diff
@@ -1027,7 +1027,10 @@ def _render_ci_provider(
             if (
                 channel_target.startswith("conda-forge ")
                 and provider_name == "github_actions"
-                and not forge_config["github_actions"]["self_hosted"]
+                and (
+                    (not forge_config["github_actions"]["self_hosted"])
+                    or (os.path.basename(forge_dir) not in SERVICE_FEEDSTOCKS)
+                )
```
Comment (beckermr) [inline conda_smithy/configure_feedstock.py:1033]:
```suggestion
                and not (
                    (forge_config["github_actions"]["self_hosted"])
                    or (os.path.basename(forge_dir) in SERVICE_FEEDSTOCKS)
                )
```

**Comment by beckermr** [comment]
pre-commit.ci autofix

**Review by isuruf — APPROVED** [review]

**Pull request merged**